### PR TITLE
Documentations reorganization

### PR DESCRIPTION
**Pull request by anchen1011**
This PR contains the improved and reorganized documentation reflecting the final decision of customer experience.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# MXNet Lambda
+
+## Introduction
+
+MXNet Lambda provides a toolkit for testing, deploying and serving deep learning models using MXNet using a serverless approach.
+
+It is built on MXNet and AWS Lambda, but could also deploy models developed with other frameworks that support ONNX (PyTorch, CNTK, Caffe2, etc.). It provides a charge-by-query solution for AWS customers who host/use AI applications, with reasonable performance and effortless and limitless scalability.
```
Comment (nswamy) [inline README.md]: MXNet -> Apache MXNet (incubating)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+HTTP/1.1 200 OK
+Content-Type: application/json
+Content-Length: 35
+{"category": "n02123159 tiger cat"}
+```
+
+## Quickstart
+
+#### Prerequisites
+AWS CLI (https://aws.amazon.com/cli/) configured
```
Comment (nswamy) [inline README.md]: 👍

File: docs/Discussion.md [hunk 3]
```diff
@@ -33,8 +19,16 @@ Other Features:
 
 1. Decent amortized runtime, but the first execution in a while is time consuming due to Lambda’s “probably start a new instance” feature
 
-## Challenges and solutions
+## Other Useful Services
+
+[Sagemaker](https://aws.amazon.com/cn/sagemaker/) is a stable AWS official tool to Build, train, and deploy machine learning models at scale.
```
Comment (nswamy) [inline docs/Discussion.md]: like mentioned in other PR please use words from official documentation for all AWS Services that you talk about (including AWS SAM)

File: docs/Discussion.md [hunk 4]
```diff
@@ -33,8 +19,16 @@ Other Features:
...
 1. Decent amortized runtime, but the first execution in a while is time consuming due to Lambda’s “probably start a new instance” feature
 
-## Challenges and solutions
+## Other Useful Services
+
+[Sagemaker](https://aws.amazon.com/cn/sagemaker/) is a stable AWS official tool to Build, train, and deploy machine learning models at scale. 
+
+[SAM-CLI](https://github.com/awslabs/aws-sam-cli) is a stable AWS Labs official tool that manages Serverless applications written with AWS Serverless Application Model (SAM). SAM CLI can be used to test functions locally, start a local API Gateway from a SAM template, validate a SAM template, fetch logs, generate sample payloads for various event sources, and generate a SAM project in your favorite Lambda Runtime.
+
+[MMS](https://github.com/awslabs/mxnet-model-server) is an AWS Labs official tool that serves deep learning models exported from MXNet or the Open Neural Network Exchange (ONNX).
```
Comment (nswamy) [inline docs/Discussion.md]: same look up official doc

File: docs/Overview.md [hunk 5]
```diff
@@ -36,11 +32,11 @@ lambda-inference-toolkit/
...
 
-LIT Base Package is a base package with decent modularization that runs a sample Lambda enabled inference task. LIT Model Bucket is an online S3 resource that host models for Lambda enabled inference. With pre-packed LIT Base Package (with MxNet pre-configured), users only need to customize local inference logics to create their own deployable Lambda enabled inference package. LIT Base Package also communicate LIT Model Bucket when being served on AWS Lambda. LIT Client is a command line tool based on AWS CLI to significantly simplify the process of construct, maintain, deploy, and update a Lambda enabled inference package. 
+Model Archive Support Framework is a framework that supports consuming Model Archive formatted machine learning models on AWS Lambda. Base Code for Running Inference on AWS Lambda is a base package with decent modularization that runs a sample Lambda enabled inference task. SAM Templates Support is AWS Serverless Application Model descriptive code to significantly simplify the process of construct, maintain, test, deploy, and update a Lambda enabled inference package. Utilities provides scripts that help process Model Archives.
```
Comment (nswamy) [inline docs/Overview.md]: Model Archive Support Framework -> Model Archive is a framework that supports consuming machine learning models

File: docs/Discussion.md (hunk 3 above)
Comment (anchen1011) [reply docs/Discussion.md]: Yep! All these words are from official website.

File: README.md (hunk 1 above)
Comment (anchen1011) [reply README.md]: Modified accordingly.

File: docs/Overview.md (hunk 5 above)
Comment (anchen1011) [reply docs/Overview.md]: Modified accordingly

**Pull request merged**